src/graphql/HolofuelAcceptOfferMutation.gql

```diff
@@ -2,7 +2,7 @@ mutation HolofuelAcceptOfferMutation ($transactionId: ID) {
   holofuelAcceptOffer(transactionId: $transactionId) {
     id
     amount
-    counterparty
+    counterparty { id, nickname }
```

Comment:
what do we use this value for?

Reply:
We use the nickname (or if not available, the id) when displaying the transaction we just edited and before the new `list_pending` data is returned.